wrapper task fails when the version 5.1 is specified.

### Current Behavior

wrapper task fail when the version 5.1(5.1-rc-1/5.1-rc-2) is specified.
\* When the version 5.0 is specified, an error does not occur.

```
.\gradlew.bat wrapper --gradle-version=5.1-rc-2
> Task :wrapper FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':wrapper'.
> Failed to write wrapper JAR to C:\Users\xxx\gradle\wrapper\gradle-wrapper.jar

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 2s
1 actionable task: 1 executed
```

### Your Environment
WIndows 10

@wolfs The below cases work differently with the same command.

This case is success.
```
.\gradlew.bat wrapper --gradle-version=5.1-rc-2
Downloading https://services.gradle.org/distributions/gradle-5.0-bin.zip
.................................................................................
Starting a Gradle Daemon, 1 incompatible and 1 stopped Daemons could not be reused, use --status for details

BUILD SUCCESSFUL in 44s
```

This case is failure.
```
.\gradlew.bat wrapper --gradle-version=5.1-rc-2
Downloading https://services.gradle.org/distributions/gradle-5.1-rc-2-bin.zip
.................................................................................
Starting a Gradle Daemon, 1 stopped Daemon could not be reused, use --status for details
> Task :wrapper FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':wrapper'.
> Failed to write wrapper JAR to C:\Users\xxx\gradle\wrapper\gradle-wrapper.jar

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 45s
1 actionable task: 1 executed
```

--- a/subprojects/build-init/src/main/java/org/gradle/api/tasks/wrapper/Wrapper.java
+++ b/subprojects/build-init/src/main/java/org/gradle/api/tasks/wrapper/Wrapper.java
@@ -17,6 +17,7 @@
 package org.gradle.api.tasks.wrapper;
 
 import com.google.common.collect.ImmutableList;
+import com.google.common.io.ByteStreams;
 import org.gradle.api.DefaultTask;
 import org.gradle.api.GradleException;
 import org.gradle.api.Incubating;
@@ -41,16 +42,15 @@ import org.gradle.wrapper.WrapperExecutor;
 import javax.annotation.Nullable;
 import javax.inject.Inject;
 import java.io.File;
+import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
+import java.io.OutputStream;
 import java.net.URL;
-import java.nio.file.Files;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Locale;
 import java.util.Properties;
-
-import static java.nio.file.StandardCopyOption.REPLACE_EXISTING;
 
 /**
  * <p>Generates scripts (for *nix and windows) which allow you to build your project with Gradle, without having to
@@ -141,8 +141,8 @@ public class Wrapper extends DefaultTask {
         if (jarFileSource == null) {
             throw new GradleException("Cannot locate wrapper JAR resource.");
         }
-        try (InputStream stream = jarFileSource.openStream()) {
-            Files.copy(stream, destination.toPath(), REPLACE_EXISTING);
+        try (InputStream in = jarFileSource.openStream(); OutputStream out = new FileOutputStream(destination)) {
+            ByteStreams.copy(in, out);
         } catch (IOException e) {
             throw new UncheckedIOException("Failed to write wrapper JAR to " + destination, e);
         }
@@ -229,7 +229,7 @@ public class Wrapper extends DefaultTask {
     public File getPropertiesFile() {
         File jarFileDestination = getJarFile();
         return new File(jarFileDestination.getParentFile(), jarFileDestination.getName().replaceAll("\\.jar$",
-                ".properties"));
+            ".properties"));
     }
 
     /**

